chardet/langgreekmodel.py

```diff
@@ -25,201 +25,192 @@
 # 02110-1301  USA
 ######################### END LICENSE BLOCK #########################
 
-# 255: Control characters that usually does not exist in any text
-# 254: Carriage/Return
-# 253: symbol (punctuation) that does not belong to word
-# 252: 0 - 9
+Windows_1253_Greek_char_to_order_map = (
```

Comment:
Why have the character-to-order maps changed?  It seems to me that you would only need to update the language model itself.